packages/terra-data-grid/tests/wdio/data-grid-spec.js

```diff
@@ -231,6 +231,35 @@ Terra.describeViewports('DataGrid', ['medium', 'large'], () => {
       Terra.validates.element('focusable-textarea-cell-trap-focus', { columnResizeSelector });
       expect(browser.$$('textarea:focus')).toBeElementsArrayOfSize(1);
     });
+
+    it('validates that keyboard inputs will not move focus from an input field', () => {
+      browser.$('#input-cell').click();
+
+      browser.keys(['ArrowRight', 'ArrowLeft']);
+
+      Terra.validates.element('data-grid-focusable-input-retains-focus', { columnResizeSelector });
+      expect(browser.$('#input-cell').isFocused());
```

Comment:
This needs a matcher:

```suggestion
      expect(browser.$('#input-cell').isFocused()).toBe(true);
```